binary/envconfig.json

```diff
@@ -0,0 +1,26 @@
+[{
+    "version": "1.0.0", 
```

Comment:
We should make all versions of format x.y (Instead of x.y.z) and also start at 2.0 version because 1.0.0 is taken: https://hub.docker.com/layers/fission/python-env-2.7/1.0.0/images/sha256-04eeeb18a2539e7e62611c49ce7493ff84a76bdc77d70fb0bad7b7e37b58ce79?context=repo

Reply:
Good point! Should we jump to 2.0.x or bump up by a minor version for respective environments? So taking Python for example, we can have 1.12.